src/dex-ui/components/DashboardHeader.tsx

```diff
@@ -0,0 +1,52 @@
+import { Text, Flex, HStack, Button } from "@chakra-ui/react";
+import { Link as ReachLink, useNavigate } from "react-router-dom";
+import { Breadcrumb, ArrowLeftIcon, Color, HashScanLink, HashscanData, Tag, StarIcon } from "@dex-ui-components";
+import { DAOType } from "@services";
+
+interface DashboardHeaderProps {
+  daoAccountId: string;
+  name: string;
+  type: DAOType;
+}
+
+export function DashboardHeader(props: DashboardHeaderProps) {
+  const { daoAccountId, name, type } = props;
+  const navigate = useNavigate();
+
+  return (
+    <Flex bg={Color.White_02} direction="column" padding="24px 80px 16px">
+      <Flex bg={Color.White_02} direction="row" gap="4">
+        <Flex bg={Color.White_02} direction="column" gap="2">
+          <HStack>
+            <Text textStyle="h3 medium">{name}</Text>
+            <Tag label={type + " DAO"} />
```

Comment:
Better if we use ` label={`${type} DAO`} `

Reply:
Changed it.